data/korath/korath hails.txt

```diff
@@ -0,0 +1,1112 @@
+		"aga' tisisis is"              # i educated in
+		"aga' lavanref"                # i taught
+		"aga' rataf ramibfaleket"      # i ate the skill
+		"aga' rataf raopakenreb ohi"   # i ate mother's knowledge
+		"kimot paga ilehfef aga'"      # father's skin gave me
+		"aga' rataf lek'af"            # i ate a master's
+
+phrase "efreti: i learned | practiced | taught"
+	word
+		"osa hanek pawaneb"            # i learned
+		"adam ohak"                    # i figured out
+		"ipawanecit it"                # i studied in
+		"adam hikanpeb"                # i practiced
+		"adam sititit it"              # i educated in
+		"adam pawalref"                # i taught
+		"adam rasaf ranicfapekes"      # i ate the skill
+		"adam rasaf raohakelrec obi"   # i ate mother's knowledge
+		"kinos hada ipebfef adam"      # father's skin gave me
+		"adam rasaf pekmaf"            # i ate a master's
+
+# Korath probably don't have this exact music and art, but I chose
+# music with a low pitch, so they'd be able to hear it well.
+
+phrase "exile: artistic or musical skill"
+	word
+		"ramibfaleket riafleh ummed"         # cello playing skills
+		"katah 'ah fols"                     # bass drumming talent
+		"katah ahok nrag laom a'aih"         # terrific tuba talent
+		"i'i'ubfut ruu'ahalkrut nrag tilaref"  # catchy contrabassoon composition
+		"raigragr ratulunnrek aimunruf"      # Mongolian throat singing
+		"zavvakat nrag nrimab mibfalek"      # the most skilled katajjaq ( Intuit throat singing)
+		"Far Upugf Enrel Ugt"                # Nam Myoho Renge Kyo (Buddhist religious chant)
+		"Fu A Al Ab Ad Ar Pips"              # Om A Ra Pa Ca Na Dhih (Buddhist religious chant)
+		"i'i'ubfut onam dimesepdg'b-k'ub"    # post-psychedelic song composition
+		"abaleheh ukle'rut uhu"              # several oboe concertos
+		"nrokab 'islo'ha tikerit"            # kinetic absurdist sculpture
+		"ralito ivanlen lebip 'ik'imael"     # hyper-realistic chainsaw carving
+		"nrokab ranravrameket tiulep"        # heroic nudity sculpting
+		"nrokab tulah nrag 'ifaris"          # dynamic boroque sculpture
+		"nrokab ire' parak 'irlesuf-k'ub"    # post-modernist land art sculpture
+		"agan ra'itom ef'irui'elbfi talk'ha" # abstract impressionism painting style
+		"lekaek tikimub ti'amtuer rlesuf"    # modern neoclassical political theatre
+		"i'tolk'rutelef aiseffud ekla'mmes"  # reconstructed (as in recovered from loss) commedia dell'arte
+		"aile'ife' alebu pasrilek"           # the most beautiful opera semiseria
+		"railak jikakelblekri inumuluekef"   # meteorological interpretive dance
+
+phrase "efreti: artistic or musical skill"
+	word
+		"ranicfapekes riafpeb unnev"         # cello playing skills
+		"kasab mmab fopt"                    # bass drumming talent
+		"kasab abok lrad paon amaib"         # terrific tuba talent
+		"imimucfus ruummabapkrus lrad siparef"  # catchy contrabassoon composition
+		"raidradr rasupullrek ainulruf"      # Mongolian throat singing
```

Comment:
I think "Mongolian" could be a little too specific, just throat singing might be better, however it could be left alone since it's another language and could just be wrote off as being a random place the korath have and only really matters in the context of text files.

Reply:
Actually, they also mention Inuit throat singing. Throat singing has different genres; each culture does it differently.

Reply:
I corrected the name of the genre in the document; it should be khoomei. (Well, actually Khöömei, but we have to stick to 7 bit ascii.) Also, eefing is in there now.

Reply:
That's extra awesome because I'm a fan of khooomei :-D